.github/workflows/make_wheel_Linux_x86.sh

```diff
@@ -7,9 +7,27 @@ rm -rf /usr/share/dotnet &
 
 export TF_NAME='tensorflow'
 
+export TF_SHORT_VERSION=$(echo $TF_VERSION | cut -d '.' -f1,2)
+
 # if tensorflow version >= 2.6.0 and <= 2.15.9
-if [[ "$TF_VERSION" =~ ^2\.([6-9]|10|11|12|13|14|15)\.[0-9]+$ ]] ; then
-  export BUILD_IMAGE="tfra/nosla-cuda11.2.1-cudnn8-ubuntu20.04-manylinux2014-python$PY_VERSION"
+if [[ "$TF_VERSION" =~ ^2\.(16)\.[0-9]+$ ]] ; then
+  export BUILD_IMAGE="tensorflow/build:$TF_SHORT_VERSION-python$PY_VERSION"
```

Comment:
No, TFRA never used the self-build TF but self-maintained CI docker; actually, the TF community provided the special docker for SIG repo, but it can't be used for TFRA for a long time because of some compilation issues. I've tried this time. It works, so I move to it.